package.json

```diff
@@ -21,7 +22,7 @@
     "grunt-mocha-phantomjs": "0.7.0",
     "grunt-usemin": "^2.0.2",
     "matchdep": "~0.3.0",
-    "markdown": "^0.5.0"
+    "marked": "*"
```

Comment:
@diegopoza could we pin to latest ? I though the `npm i --save` did it automatically.

oh! I see... it did on the dependencies :point_up: